### ISSUE TITLE
Type error TS2769 when calling `.text`.

### ISSUE THREAD

**Issue by ByMsx** [grammyjs/menu#11]
Hi, dear developer! Thank you for your great library. 

My code:
```javascript
export const adminMenu = new Menu<MyContext>('adminMenu')
  .text('text1', async (ctx) => { // throws a type error
    // ...
  })
  .row()
  .submenu('text2', 'chatsList') // throws a type error
  .submenu(() => 'text3', 'urlWhitelist'); // passes type checking
```

Here is a type error:
```
error TS2769: No overload matches this call.
  Overload 1 of 3, '(text: DynamicString<MyContext>, menu: string, ...middleware: MenuMiddleware<MyContext>[]): Menu<MyContext>', gave the following error.
    Argument of type 'string' is not assignable to parameter of type 'DynamicString<MyContext>'.
  Overload 2 of 3, '(text: TextAndPayload<MyContext>, menu: string, ...middleware: MenuMiddleware<Context & MenuFlavor & ChatSettingsFlavor & UserStatusFlavor & SessionFlavor<...> & { ...; }>[]): Menu<...>', gave the following error.
    Argument of type 'string' is not assignable to parameter of type 'TextAndPayload<MyContext>'.
  Overload 3 of 3, '(text: MaybePayloadString<MyContext>, menu: string, ...middleware: MenuMiddleware<MyContext>[]): Menu<MyContext>', gave the following error.
    Argument of type 'string' is not assignable to parameter of type 'MaybePayloadString<MyContext>'.
```

According docs (https://grammy.dev/plugins/menu.html#introduction) I can use simple `string` as first argument of `.text` or `.submenu`. 

P.S. Sorry for my bad English.

**Comment by KnorpelSenf** [comment]
Thanks for the report! I can reproduce the issue.

**Comment by KnorpelSenf** [comment]
The fix for this issue was shipped in version 1.0.1.
